pyproject.toml

```diff
@@ -20,6 +20,8 @@ motor = "^2.5"
 click = ">=7"
 toml = "*"
 yarl = ">=1.6"
+redis = "^4.3.1"
```

Comment:
it would be great to have these guys as extra dependencies: https://python-poetry.org/docs/pyproject#extras